var/spack/repos/builtin/packages/py-diskcache/package.py

```diff
@@ -10,6 +10,9 @@ class PyDiskcache(PythonPackage):
     homepage = "http://www.grantjenks.com/docs/diskcache/"
     pypi = "diskcache/diskcache-4.1.0.tar.gz"
 
+    version('5.2.1', sha256='1805acd5868ac10ad547208951a1190a0ab7bbff4e70f9a07cde4dbdfaa69f64')
     version('4.1.0', sha256='bcee5a59f9c264e2809e58d01be6569a3bbb1e36a1e0fb83f7ef9b2075f95ce0')
 
+    depends_on("python", type=("build", "run"), when="@:4.1.0")
+    depends_on("python@3:", type=("build", "run"), when="@5.2.1:")
```

Comment:
The latest release still seems to support Python 2. Can you remove these lines?

Reply:
I see this line in [setup.py](https://github.com/grantjenks/python-diskcache/blob/v5.2.1/setup.py)

`python_requires='>=3'`

Reply:
Ah yes, my mistake, ignore me.